BrainPortal/app/controllers/userfiles_controller.rb

```diff
@@ -546,9 +546,13 @@ def create #:nodoc:
         userfile.errors.each do |field, error|
           flash[:error] += "#{field.to_s.capitalize} #{error}.\n"
         end
+        flash[:error]  += "It might help to rename the file.\n" if flash[:error].include? "Name is already"
```

Comment:
The error message is normally `Name has already been taken.` which doesn't match `Name is already`

Reply:
corrected as requested, but feel free to drop that line if still in doubts